Changes.md

```diff
@@ -28,6 +28,7 @@ Fixes
   - Fixed bug in `Key.setType()`. Previously it modified the value instead of the type.
   - Fixed crash when dragging multiple keys around in editor.
   - Fixed bug which resulted in preview key being incorrectly drawn at the origin.
+  - Fixed bug when pressing "f" to frame viewport which resulted in a blank background in certain cases.
```

Comment:
Couple of little tweaks please :

- Use `<kbd>F</kbd>`, since it gives nice formatting in the docs, like so : <kbd>F</kbd>.
- Include the associated issue number at the end : `in certain cases (#4410)`.